.github/scripts/map-users/main.mjs

```diff
@@ -12,13 +12,8 @@ export class Mapper {
   requestedReviewers() {
     let users = [];
 
-    for (let user of this.payload?.pull_request?.requested_reviewers) {
-
-      const login = user?.login;
-      if (login === undefined) {
-        continue;
-      }
-
+    const login = this.payload?.requested_reviewer?.login;
+    if (login !== undefined) {
```

Comment:
`undefined is not a function`

Reply:
true … truthy? … or something like that: 
![image](https://github.com/trustification/trustify/assets/202474/0a4e2a38-d085-4856-8674-686a28b8d885)